src/controllers/userController.js

```diff
@@ -0,0 +1,69 @@
+const {
+  getUsers,
+  getUserById,
+  create,
+  remove,
+  update,
+} = require('../services/userService');
+
+const getAll = (req, res) => {
+  res.statusCode = 200;
+  res.send(getUsers());
```

Comment:
The `getUsers` function is called synchronously. If it involves database access, consider using asynchronous operations to prevent blocking the event loop.